### PR TITLE
refactor(example): updated with the latest version of the clipboard API

### PR DESCRIPTION
**Pull request by tbor00**
Refactor copyToClipBoard function with the latest version of clipboard API

### CODE REVIEW

**Comment by dai-shi** [comment]
@rnike Can you review this please?

**Review by rnike — CHANGES_REQUESTED** [review]

File: examples/src/utils/copy-to-clipboard.js [hunk 1]
```diff
@@ -1,12 +1,3 @@
...
-  el.style.position = 'absolute'
-  el.style.left = '-9999px'
-  document.body.appendChild(el)
-  el.select()
-  // FIXME: might need to update this with the clipboard API in the future
-  document.execCommand('copy')
-  document.body.removeChild(el)
-}
+ return navigator.clipboard.writeText(str)
+}
```
[the remark this answers is not in the document]
Comment (dai-shi) [reply examples/src/utils/copy-to-clipboard.js:3]: looks like we don't run prettier for examples in CI.

https://github.com/pmndrs/zustand/blob/f949d04ff29eccbf2d27661c0d5c13b34ecee04d/package.json#L71-L72

we can add `examples` there.

[the remark this answers is not in the document]
Comment (rnike) [reply examples/src/utils/copy-to-clipboard.js:3]: okay, maybe we can have it later, I have no more questions here.

Comment (dai-shi) [inline examples/src/utils/copy-to-clipboard.js:3]:
```suggestion
  return navigator.clipboard.writeText(str)
}
```

[the remark this answers is not in the document]
Comment (dai-shi) [reply examples/src/utils/copy-to-clipboard.js:3]: yeah, let's try it later.

**Review by dai-shi — APPROVED** [review]
@tbor00 thanks for your contribution.
@rnike thanks for your review.

File: examples/src/components/CopyButton.js [hunk 2]
```diff
@@ -9,10 +9,12 @@ export default function CopyButton({ code, ...props }) {
   const [isCopied, setIsCopied] = useState(false)
 
   const handleCopy = () => {
-    copyToClipboard(code)
-    setIsCopied(true)
-    setTimeout(() => setIsCopied(false), 3000)
+    copyToClipboard(code).then(() => {
+      setIsCopied(true)
+      setTimeout(() => setIsCopied(false), 3000)
```
[the remark this answers is not in the document]
Comment (tbor00) [reply examples/src/components/CopyButton.js:14]: Yes, i thing what i can create a new issue and woking for this. Thanks

**Pull request merged**